src/main/java/com/google/api/generator/gapic/composer/MethodSampleCodeHelperComposer.java

```diff
@@ -82,6 +87,49 @@ public static TryCatchStatement composeUnaryRpcMethodSampleCode(
         .build();
   }
 
+  public static TryCatchStatement composeUnaryPagedRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames,
+      TypeNode pagedItemType) {
+    VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
+    // Assign each method arguments with its default value.
+    Map<String, VariableExpr> methodArgVarExprMap = createMethodArgumentsVariableExprs(arguments);
+    List<Expr> methodArgumentsAssignmentExpr =
+        assignMethodArgumentsWithDefaultValues(arguments, methodArgVarExprMap, resourceNames);
+    List<Expr> methodArgVarExprs =
+        arguments.stream()
+            .map(arg -> methodArgVarExprMap.get(arg.name()))
+            .collect(Collectors.toList());
+    // For loop client on iterateAll method.
+    // e.g. for (LogEntry element : loggingServiceV2Client.ListLogs(parent).iterateAll()) {
+    //          //doThingsWith(element);
+    //      }
+    VariableExpr elementVarExpr = createVariableExpr("element", pagedItemType);
+    Expr clientMethodIterateAllExpr =
+        createClientMethodIteratorAllExpr(
+            clientVarExpr, method.name(), methodArgVarExprs, pagedItemType);
+    ForStatement loopIteratorStatement =
+        ForStatement.builder()
+            .setLocalVariableExpr(elementVarExpr.toBuilder().setIsDecl(true).build())
+            .setCollectionExpr(clientMethodIterateAllExpr)
+            .setBody(Arrays.asList(createLineCommentStatement("doThingsWith(element);")))
+            .build();
+
+    List<Expr> bodyExpr = new ArrayList<>();
+    bodyExpr.addAll(methodArgumentsAssignmentExpr);
+    List<Statement> bodyStatements =
+        bodyExpr.stream().map(e -> ExprStatement.withExpr(e)).collect(Collectors.toList());
```

Comment:
Why not stream `methodArgumentsAssignmentExpr` directly, so we don't have to create another List?

Reply:
Discussed in #573